hono/src/services/items.ts

```diff
@@ -40,20 +41,19 @@ app.post(
   injectDB,
   async (c) => {
     const body = c.req.valid('json')
-
-    return c.json(
-      (
-        await c
-          .get('db')
-          .insert(items)
-          .values({
-            ...body,
-            authorUsername: 'rak3rman', // TODO: remove and replace with author integration
-          })
-          .onConflictDoNothing()
-          .returning()
-      )[0]
-    )
+    const res = (
+      await c
+        .get('db')
+        .insert(items)
+        .values({
+          ...body,
+          authorUsername: 'rak3rman', // TODO: remove and replace with author integration
+        })
+        .onConflictDoNothing()
```

Comment:
Why do we do nothing here? Should we notify the user that the item wasn't added because it already exists?

Reply:
Good point, I'll look into a way to catch and return an error here.